release-team/role-handbooks/enhancements/README.md

```diff
@@ -301,6 +302,8 @@ See the sig-architecture Enhancements [KEP Template](https://github.com/kubernet
 
 **All** enhancements going into the release must have their pull requests in a merge-ready state with approved and lgtm labels applied. This includes tests, and docs PRs opened by the due date. Due to the Prow queue, this may mean that approved PRs merge shortly after the deadline.
 
+> **Note**: KEPs that contain bug fixes as part of their implementation work (even if the primary work is fixing existing issues rather than adding new features) are perfectly acceptable to mark as `Tracked for code freeze` for the purposes of enhancements tracking. The key requirement is that all code changes are properly reviewed and approved, regardless of whether they introduce new functionality or fix existing issues.
```

Comment:
Just to clarify, do you mean that KEPs that *only* contain bug fixes should still be tracked? I think this goes against the messaging from https://groups.google.com/g/kubernetes-sig-release/c/LTzDa7OyBSA/m/IrKW973HAQAJ which says that bug fixes don't need to be tracked